runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -67,13 +67,15 @@ final class GrpcClientSettings private (
   val overrideAuthority: Option[String] = None,
   val trustedCaCertificate: Option[String] = None,
   val deadline: Duration = Duration.Undefined,
-  val userAgent: Option[String] = None) {
+  val userAgent: Option[String] = None,
+  val useTls: Boolean = true) {
```

Comment:
We could also implement `useTls` here as `def useTls = trustedCaCertificate.isDefined`?

I think this will be removed/replaced in #266 anyway so :+1:

Reply:
Oh, I was under the impression that it already worked with the default keychain if you don't specify and the trusted is just for self signed?